app/styles/tasks.css

```diff
@@ -182,6 +182,19 @@
   justify-content: space-between;
   align-items: center;
 }
+.task-details__title-container a {
+  max-width: 66%;
+}
+@media screen and (min-width: 900px) {
+  .task-details__title-container a {
+    max-width: 66%;
+  }
+}
+@media screen and (max-width: 900px) {
+  .task-details__title-container a {
+    max-width: 50%;
+  }
+}
```

Comment:
- please use non percentage units as this will vary for different pixel density and screen sized

Reply:
Fixed

Reply:
done